src/backend/aspen/workflows/import_lineages/load_lineages.py

```diff
@@ -0,0 +1,168 @@
+from typing import Any, List
+
+import click
+import requests
+import sqlalchemy as sa
+
+from aspen.config.config import Config
+from aspen.database.connection import (
+    get_db_uri,
+    init_db,
+    session_scope,
+    SqlAlchemyInterface,
+)
+from aspen.database.models import Pathogen, PathogenLineage
+from aspen.util.pathogen_configs import get_lineage_urls
+from aspen.workflows.shared_utils.database import (
+    create_temp_table,
+    drop_temp_table,
+    mv_table_contents,
+)
+
+
+def download_lineages(
+    url: str,
+    format: str,
+    print_response: bool,
+) -> Any[str, List[str]]:
```

Comment:
I think the type hint here is just `Any`. That's the hint you wrote for `format_lineage_data()`, which is the consumer of this function's output, and I think it works fine. Especially since JSON objects are a nightmare to type in Python.

If you are certain that the two expected types are `str` and `List[str]` you can write `Union[str, List[str]]` instead. In that case you'll have to also use is as the type hint for the `source_data` parameter.

The import test worked with either type on my machine.